doc/connectivity/bluetooth/bluetooth-qual.rst

```diff
@@ -3,6 +3,21 @@
 Bluetooth Qualification
 #######################
 
+Qualification setup
+*******************
+
+The Zephyr Bluetooth host is qualified using Bluetooth's PTS (Profile Tuning Suite) software.
```

Comment:
```suggestion
The Zephyr Bluetooth host can be qualified using Bluetooth's PTS (Profile Tuning Suite) software.
```

It's been a long time since we've done a qualification, so might be a bit much to claim that is is qualified today.

Reply:
True, but that is so long ago, that the current host and the qualified host are hardly the same host subsystem anymore :|